sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/campaigndata/CampaignDataView.java

```diff
@@ -50,21 +61,78 @@
 	private final CampaignDataGrid grid;
 	private CampaignFormDataFilterForm filterForm;
 
+	protected OptionGroup campaignFormElementImportance;
+
+	public static final String ONLY_IMPORTANT_FORM_ELEMENTS = "onlyImportantFormElements";
+
 	public CampaignDataView() {
 		super(VIEW_NAME);
 
 		criteria = ViewModelProviders.of(getClass()).get(CampaignFormDataCriteria.class);
 		grid = new CampaignDataGrid(criteria);
 
 		VerticalLayout mainLayout = new VerticalLayout();
-		mainLayout.addComponent(createFilterBar());
+		HorizontalLayout filtersLayout = new HorizontalLayout();
+
+		filtersLayout.setWidthFull();
+		filtersLayout.setMargin(false);
+		filtersLayout.setSpacing(true);
+
+		CampaignFormDataFilterForm filterBar = createFilterBar();
+		filtersLayout.addComponent(filterBar);
+		filtersLayout.setComponentAlignment(filterBar, Alignment.TOP_LEFT);
+		filtersLayout.setExpandRatio(filterBar, 0.8f);
+
+		OptionGroup importanceFilterLayout = createImportanceFilterSwitch();
+		filtersLayout.addComponent(importanceFilterLayout);
+		filtersLayout.setComponentAlignment(importanceFilterLayout, Alignment.TOP_RIGHT);
+		filtersLayout.setExpandRatio(importanceFilterLayout, 0.2f);
+
+		mainLayout.addComponent(filtersLayout);
+
+		filterForm.getField(CampaignFormDataCriteria.CAMPAIGN_FORM_META).addValueChangeListener(e -> {
+			Object value = e.getProperty().getValue();
+			if (value == null) {
+				campaignFormElementImportance.setEnabled(false);
+				campaignFormElementImportance.setValue(CampaignFormElementImportance.IMPORTANT);
+			} else {
+				campaignFormElementImportance.setValue(CampaignFormElementImportance.IMPORTANT);
+				campaignFormElementImportance.setEnabled(true);
+			}
+		});
+
+		campaignFormElementImportance.addValueChangeListener(e -> {
+			grid.reload();
+			createFormMetaChangedCallback()
+				.accept((CampaignFormMetaReferenceDto) filterForm.getField(CampaignFormDataCriteria.CAMPAIGN_FORM_META).getValue());
+		});
+
 		mainLayout.addComponent(grid);
 		mainLayout.setMargin(true);
 		mainLayout.setSpacing(false);
 		mainLayout.setSizeFull();
 		mainLayout.setExpandRatio(grid, 1);
 		mainLayout.setStyleName("crud-main-layout");
 
+		if (UserProvider.getCurrent().hasUserRight(UserRight.CAMPAIGN_FORM_DATA_EXPORT)) {
+			VerticalLayout exportLayout = new VerticalLayout();
+			{
+				exportLayout.setSpacing(true);
+				exportLayout.setMargin(true);
+				exportLayout.addStyleName(CssStyles.LAYOUT_MINIMAL);
+				exportLayout.setWidth(250, Unit.PIXELS);
+			}
+
+			PopupButton exportPopupButton = ButtonHelper.createIconPopupButton(Captions.export, VaadinIcons.DOWNLOAD, exportLayout);
+			addHeaderComponent(exportPopupButton);
+
+			{
+				StreamResource streamResource =
+					new GridExportStreamResource(grid, "sormas_campaign_data", createFileNameWithCurrentDate("sormas_campaign_data_", ".csv"));
```

Comment:
Use the ignoredPropertyIds parameter of GridExportStreamResource to hide the edit column of the grid in the export.